lib/rack/error_handler.rb

```diff
@@ -0,0 +1,23 @@
+module Rack
+  class ErrorHandler
```

Comment:
Similar to `BadRequest`, unless this handles something other than multipart errors, it should be moved under the `Multipart` namespace.

Also, there doesn't appear to be any specs for this.

Reply:
It's currently handling ALL errors which aren't `Rack::BadRequest` and mapping them to status=500.

I haven't written specs because it's just an example of what we can do.

Reply:
Added some specs.